AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/hearing-details/hearing-details.component.spec.ts

```diff
@@ -142,6 +155,23 @@ describe('HearingDetailsComponent', () => {
         const result = component.getParticipantInfo('123-1234');
         expect(result).toBe('');
     });
+    fdescribe('feature flag turn on and off ', () => {
+        fit('should not show allocated to if work allocation feature flat is off', async () => {
```

Comment:
remove focus fdescribe and fit -> set it back to describe it

Reply:
changed the fdescribe.

Reply:
Not quite.. fit aswell

Reply:
fixed